src/components/Edge.vue

```diff
@@ -86,7 +86,7 @@ export default {
       }
     },
     bezierOffset () {
-      return Math.abs(0.5 * (this.centerEnd.x - this.centerStart.x))
+      return Math.tanh(Math.abs(0.5 * (this.centerEnd.x - this.centerStart.x)) / this.padding) * this.padding
```

Comment:
This offset will now be unable to surpass the padding

Reply:
I still feel this is undesirable, as vertically aligned connectors should still produce a spline instead of a straight line. This improves visibility of the edges and avoids weird movement when dragging the end from the left to the right or vice-versa.

Reply:
How about a minimum offset?

Reply:
What behaviour do you invision? Will a dynamic offset even suit your needs? Any other ideas for a function?

Reply:
This is good, I approve.